.github/workflows/pr.yaml

```diff
@@ -16,16 +16,28 @@ jobs:
   pr-builder:
     needs:
       - checks
-      - tests
+      - build-conda
+      - test-conda
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.10
   checks:
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/checks.yaml@branch-24.10
     with:
       enable_check_generated_files: false
-  tests:
-    name: Run build and tests
+  build-conda:
+    name: Build Conda Python
     needs: checks
     secrets: inherit
-    uses: ./.github/workflows/build.yaml
+    uses: ./.github/workflows/conda-python-build.yaml
+    with:
+      build_type: pull-request
+      matrix_filter: map(select(.CUDA_VER == "12.8.0" and .PY_VER == "3.11"))
+  test-conda:
+    name: Test Conda Python
+    needs: build-conda
+    secrets: inherit
+    uses: ./.github/workflows/conda-python-test.yaml
+    with:
+      build_type: pull-request
+      matrix_filter: map(select(.CUDA_VER == "12.8.0" and .PY_VER == "3.11"))
```

Comment:
I recommend avoiding hardcoded versions in filters. Use a filter that uses "latest CUDA and Python", something like https://github.com/rapidsai/cudf/blob/293ac5d86b3ae849ea3ce8764a27cc67028410b6/.github/workflows/pr.yaml#L255-L256

You can find example filters for a lot of use cases with this search query: https://github.com/search?q=org%3Arapidsai+%2F%5Cs%2Bmatrix_filter%3A%2F&type=code

Reply:
Nice - thanks for the pointer that looks like a great filter for PRs.